### PR TITLE
fix logo page

### PR DESCRIPTION
**Pull request by alexcontini**
@lumjjb here is a PR to update the page with the changes. I wasn't sure about the spacing issue, I did remove the other colors with the codes as it was repetitive to have both sections, but let me know if I missed something or if there are any issues with the new page!

### CODE REVIEW

File: logo/README.md [hunk 1]
```diff
@@ -50,27 +51,51 @@
...
+    </tr>
+  <tr>
+        <th>#6F6F7F</th>
+        <th>#474756</th>
+        <th>#141419</th>
+        <th>#152356</th>
+        <th>#4A6CA4</th>
+        <th>#389BB2</th>
+        <th>#D81637</th>
+ 	<th>#F98903</th>
```
Comment (lumjjb) [inline logo/README.md]: nit: indentation

File: logo/README.md [hunk 2]
```diff
@@ -50,27 +51,51 @@
...
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%23474756.svg" style="width:100px"></td>       
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%23141419.svg" style="width:100px"></td>
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%23152356.svg" style="width:100px"></td>
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%234A6CA4.svg" style="width:100px"></td>
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%23389BB2.svg" style="width:100px"></td>
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%23D81637.svg" style="width:100px"></td>
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%23F98903.svg" style="width:100px"></td>
+<td><img src="https://github.com/alexcontini/sig-security/blob/contini-fix-logo-page/logo/colors/%23F7C906.svg" style="width:100px"></td>
+    </tr>
```
Comment (lumjjb) [inline logo/README.md]: nit, indentation, we can just flush all of the `<tr>` and `</tr>`s to the start of line

File: logo/README.md [hunk 3]
```diff
@@ -50,27 +51,51 @@
...
+  <tr>
+        <th>#6F6F7F</th>
+        <th>#474756</th>
+        <th>#141419</th>
+        <th>#152356</th>
+        <th>#4A6CA4</th>
+        <th>#389BB2</th>
+        <th>#D81637</th>
+ 	<th>#F98903</th>
+        <th>#F7C906</th>
```
Comment (lumjjb) [inline logo/README.md]: One color seems to be missing here: #85C2D2

File: logo/README.md.sb-d9004420-eelPlk [hunk 4]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```
Comment (lumjjb) [inline logo/README.md.sb-d9004420-eelPlk]: hmm, what's this file used for? is it necessary or can we delete it?

File: logo/README.md [hunk 5]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (lumjjb) [inline logo/README.md]: nit: remote line

**Comment by alexcontini** [comment]
* opening a new PR as I made this one very messy

**Pull request closed without merging**